src/cmd/linuxkit/run_qemu.go

```diff
@@ -547,7 +615,10 @@ func splitPublish(publish string) (publishedPorts, error) {
 }
 
 func buildQemuForwardings(publishFlags multipleFlag, containerized bool) (string, error) {
-	forwardings := "user"
```

Comment:
Correct. The caller now just unconditionally appends the result of this function to the mode, and the caller is responsible for checking that the mode is consistent with doing forwarding, which in practice means `user` (the check is in the `switch netMode[0]` bit).